.ci/scripts/run_tests_torch_ucc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eEx
+set -o pipefail
+
+command -v mpirun
+export TORCH_UCC_XCCL_TLS=ucx
+export UCX_WARN_UNUSED_ENV_VARS=n
+ucx_info -e -u t
+TORCH_UCC_SRC_DIR_WITH_XCCL_SUPPORT="${TORCH_UCC_SRC_DIR}_xccl"
+
+echo "UCC barrier"
+/bin/bash ${TORCH_UCC_SRC_DIR_WITH_XCCL_SUPPORT}/test/start_test.sh ${TORCH_UCC_SRC_DIR_WITH_XCCL_SUPPORT}/test/torch_barrier_test.py --backend=gloo
```

Comment:
why backend=gloo?

Reply:
@Sergei-Lebedev could you please comment?

Reply:
backend option sets backend to compare with, i.e. ucc collective will be compared with gloo collective for correctness check